configs/FREEAXISF4/config.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F411
+
+#define BOARD_NAME        FREEAXISF4
+#define MANUFACTURER_ID   CLNE
+
+#define USE_ACC
+#define USE_GYRO
+#define USE_ACCGYRO_BMI270
+#define USE_BARO
```

Comment:
When remove the BARO specfic driver we also should remove this line :)
```suggestion
```

Reply:
Using this define defies cloud build as we try to isolate code to get maximum space for future development. There has been a lot of effort to configure only the drivers physical present.